mysql/datadog_checks/mysql/mysql.py

```diff
@@ -447,7 +451,7 @@ def _collect_metrics(self, db, tags):
 
         if not is_affirmative(
             self._config.options.get('disable_innodb_metrics', False)
-        ) and self._is_innodb_engine_enabled(db):
+        ) and self._check_innodb_engine_enabled(db):
```

Comment:
Same as prior comment, we can directly use `self. _is_innodb_engine_enabled` once it's initialized

Reply:
as discussed on the call keeping the current version